### ISSUE TITLE
Meta tags are not being rendered correctly

### ISSUE THREAD

**Issue by divineleft** [solidjs/solid-meta#20]
In an app that uses these components to change the title or meta links and such, they're not being rendered with text or any of their properties and just display the name of the tag; example:

````
import { Title } from '@solidjs/meta'
export default function Main() {
   return <div>
    <Title> This is a title </Title>
 </div>
}
````

This code should create a title tag with the text `'This is a title'`, but in the generated html, only the tag `<title></title>` with no text, is rendered, and the same happens with all the other tags in this package, `<Meta />`, `<Link />`, etc.

I downgraded to solidjs 1.5.9 and everything works as expected, so I'm guessing this has to do with something related to Deproxification?

**Comment by ryansolid** [comment]
I believe this is fixed now in 0.28.1

**Comment by divineleft** [comment]
> I believe this is fixed now in 0.28.1

Came home to a new update and it seems to be working just fine. Great work and thanks!
